games/b.yaml

```diff
@@ -471,7 +472,8 @@
 
 - name: Buzz Aldrin's Race Into Space
   meta:
-    genre: [simulation,TBS]
+    genre: [Strategy, Simulation, TBS]
```

Comment:
There is no need for a Strategy tag along with TBS. I've made an effort to catalog all entries as either TBS or RTS and I think this is far more useful to people browsing the list than a more general `strategy`label.